fuel_test/test_env.sh

```diff
@@ -34,5 +34,9 @@ function health_check {
 }
 
 FM_IP=$(get_fm_ip "$XS_HOST" "$FM_NAME")
-RESULT=$(health_check "$FM_IP" "$ENV_NAME")
-echo $RESULT | grep -qv '[failure]'
\ No newline at end of file
+result=$(health_check "$FM_IP" "$ENV_NAME")
+RESULT=$(comm -3 \
+<(echo -e $result | egrep -o "\[failure\] '[^\']+'" | sed -e "s|\[failure\] ||g" | sed -e "s|'||g" | sort) \
+<(printf '%s\n' "${IGNORE_CHECKS[@]}" | sort))
+[[ -n $RESULT ]] && exit 1
```

Comment:
If we have some failed tests, we should at least print them to STDOUT rather than just discard them